src/app/domUtils.ts

```diff
@@ -97,18 +97,25 @@ export const scrollTo = (elem: HTMLElement | Element | string) => {
   return scrollToElement(elem, {offset: getScrollPadding()});
 };
 
-export const scrollIntoView = (elem: HTMLElement) => {
+/**
+ * This function will scroll into @param firstElement view but it will take into consideration
+ * the @param lastElement position to check if scrolling is required.
+ */
+export const scrollIntoView = (firstElem: HTMLElement, lastElem?: HTMLElement) => {
   const window = assertWindow();
-  if (!window.document.body.contains(elem)) { return; }
+  if (!window.document.body.contains(firstElem)) { return; }
```

Comment:
Maybe we should make the same check if `lastElem` is passed.

Reply:
I think we would like to scroll to the `firstElement` if the `lastElement` is not defined, since it is supposed to be used only for calculation purposes